src/containers/ProtocolScreen.js

```diff
@@ -28,13 +45,24 @@ class Protocol extends Component {
 
   // change the stage to the previous
   onClickBack = () => {
-    if (!this.props.stage.prompts || this.props.isFirstPrompt()) {
+    if (this.getInterfaceRef() && !this.getInterfaceRef().isStageBeginning()) {
+      this.getInterfaceRef().clickPrevious();
+    } else if (!this.props.stage.prompts || this.props.isFirstPrompt()) {
       this.props.changeStage(`${this.props.pathPrefix}/${this.props.previousIndex}?back`);
     } else {
       this.props.promptBackward();
     }
   }
 
+  getInterfaceRef = () => {
```

Comment:
Would be nice to document the contract for interfaces using this functionality (e.g., implement `clickLast`, `isEnding`, etc.) Maybe there's a better place, but here would work.